mne/preprocessing/maxwell.py

```diff
@@ -505,3 +533,36 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _overlap_projector(mat_1, mat_2, delta=0.02):
+    """Calculate projector for removal of subspace intersection in tSSS"""
+    # delta necessary to deal with noise when finding identical signal
+    # directions in the subspace. See the end of the Results section in [2]_
+
+    # TODO: Get comments on naming conventions from @staulu
+
+    # Compute SVD to get temporal bases. Matrices must have shape
+    # (n_samps x n_channels) when passed into svd computation
+    U_1, S_1, V_1 = svd(mat_1.T)
+    U_2, S_2, V_2 = svd(mat_2.T)
+
+    # Compute subspace intersection between temporal bases
+    # Q_1, Q_2 should have shape (n_time_pts x n_bases)
+    # R_1, R_2 should have shape (80 x 80)
+    Q_1, R_1 = np.linalg.qr(V_1.T)
+    Q_2, R_2 = np.linalg.qr(V_2.T)
```

Comment:
scipy is guaranteed to be compiled against optimized fortran code. So it's
safer.